.vscode/settings.json

```diff
@@ -19,5 +19,16 @@
 	"**/.cache/**",
 	"**/jetpack_vendor/**/jetpack_vendor/**",
 	"**/jetpack_vendor/**"
-	]
+	],
+	"phpcs.enable": true,
```

Comment:
I think this may only be useful for folks using the phpcs VS code extension, and not the `PHP_CodeSniffer` VS code extension ([reference](https://marketplace.visualstudio.com/items?itemName=obliviousharmony.vscode-php-codesniffer)).